src/Parallel/spanningtrees/boruvka-distributed.jl

```diff
@@ -0,0 +1,129 @@
+#=
+boruvka_multithread_lightgraph:
+- Julia version: 1.3.1
+- Author: hanstananda
+- Date: 2020-03-15
+=#
+@everywhere using LightGraphs, SimpleWeightedGraphs
+using CPUTime
+using DelimitedFiles
+@everywhere using Base
+@everywhere using DataStructures
+using SharedArrays
+
+"""
+    boruvka_mst_distributed(g)
+
+Return a tuple `(mst, weights)` where `mst` is a vector of edges representing the
+optimum (minimum, by default) spanning tree of a connected, undirected graph
```

Comment:
minimum by default, but how does one change the default?

Reply:
I'll soon implement the option to support the maximum as well